Jenkinsfile

```diff
@@ -30,9 +30,15 @@ node {
 
         withCredentials([usernamePassword(credentialsId: '48253a45-d82c-43c8-b39e-031c511bc475', passwordVariable: 'DOCKER_REGISTRY_PASS', usernameVariable: 'DOCKER_REGISTRY_USER')]) {
             sh "docker login --username=${DOCKER_REGISTRY_USER} --password=${DOCKER_REGISTRY_PASS} "
+             if (${BRANCH_NAME}=='master')
```

Comment:
This works? I think it is `${env.BRANCH_NAME}` cause it is not in a `sh` clause.